rma_sale/models/rma.py

```diff
@@ -17,10 +17,11 @@ class Rma(models.Model):
         readonly=True,
         states={"draft": [("readonly", False)]},
     )
-    allowed_picking_ids = fields.Many2many(
-        comodel_name="stock.picking", compute="_compute_allowed_picking_ids",
+    picking_id = fields.Many2one(
+        domain="[('state', '=', 'done'), ('picking_type_id.code', '=', 'outgoing')] "
+        "+ [('sale_id', '=', order_id)] if order_id else [] "
+        "+ [('partner_id', 'child_of', commercial_partner_id)] if partner_id else []"
```

Comment:
@pedrobaeza Are right fields really evalutated dynamically?

Reply:
I'm trying. On the worst case, we will need to repeat the domain.

Reply:
I would say if you want something more efficient, web_domain_field is the right solution

Reply:
Efficiency can be the same, and what I don't want is to depend on it. I'm seeing Pyjs (http://pyjs.org/) doesn't support ternary operators, nor list sums, but I'm working on adapting them.

Reply:
Sectarism will loose you :smiley: adding a dependency is not evil if the addition is good for the job. Hacking is

Reply:
This is not hacking. It's using the framework possibilities. Already done and working.

Reply:
Judge yourself that is very simple.

Reply:
#317 is the approach with `web_domain_field`, and apart from adding the new dependency, it's not simpler.